core/src/main/kotlin/xtdb/buffer_pool/RemoteBufferPool.kt

```diff
@@ -179,8 +185,11 @@ class RemoteBufferPool(
             memCacheMisses?.increment()
             diskCache.get(key) { k, tmpFile ->
                 diskCacheMisses?.increment()
+                networkRead?.increment(k.fileSize().toDouble())
                 objectStore.getObject(k, tmpFile)
-            }.thenApply { entry -> Pair(PathSlice(entry.path, pathSlice.offset, pathSlice.length), entry) }
+                      }.thenApply {
```

Comment:
nit: indentation gone awry here - think this should be aligned with 186?

Reply:
Sorted!